static/.htaccess

```diff
@@ -9,6 +9,11 @@ Options -Indexes -MultiViews
 # Web Modeler API
 RewriteRule ^docs/next/apis-tools/web-modeler-api/index/?(.*)$ /docs/next/apis-tools/web-modeler-api/overview/$1 [R=301,L]
 
+# Zeebe API
+RewriteRule ^docs/grpc/?$ /docs/zeebe-api/grpc/ [R=301,L]
+
+# Console API
+RewriteRule ^docs/console-api-reference/?$ /docs/console-api/console-api-reference/ [R=301,L]
```

Comment:
I don't think these rules will catch anything in this PR, and will likely lead to a failed link-check step.

The changes are all for vNext, but these redirect rules are for vCurrent. 

In fact, I think we've stated that we don't care to maintain redirect rules in vNext. These rules will be useful for #3151, when these changes make their way to v8.4, but until then I think they can be removed.